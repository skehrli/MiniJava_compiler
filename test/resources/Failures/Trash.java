class Trash {
  this class is garbage
  Lets see what the /* compiler */ does
  // with it!
}
