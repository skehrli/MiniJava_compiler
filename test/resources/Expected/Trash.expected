CLASS
ID(Trash)
LBRACE
THIS
CLASS
ID(is)
ID(garbage)
ID(Lets)
ID(see)
ID(what)
ID(the)
ID(does)
RBRACE
